### PR TITLE
in install_raspbian.md added reference to the ziggy ai image server

### PR DESCRIPTION
**Pull request by Bulwagga**

### CODE REVIEW

File: docs/install_raspbian.md [hunk 1]
```diff
@@ -1,154 +1,217 @@
...
-## Step 3: Install OVOS-CORE
+### Step 3: Install ovos-core
 
-There are some recommendations to use a venv for ovos. This guide DOES NOT do that.  The ovos headless stack on a RPi3 is about all it can handle.  It is assumed that this is a dedicated ovos device, therefore no venv is required.
+There are some recommendations to use a venv for OVOS. This guide DOES NOT do that.  The OVOS headless stack on a RPi3 is about all it can handle.  It is assumed that this is a dedicated OVOS device, therefore no venv is required.
 
-There are a few packages required for ovos, so we will install those first
+We will be cloning code from a git repository, so before starting we need to install git.
 
-`sudo apt install build-essential python3-dev python3-pip swig libssl-dev libfann-dev portaudio19-dev libpulse-dev cmake git libncurses-dev pulseaudio-utils`
```
Comment (mikejgray) [inline docs/install_raspbian.md:122]: Are these all included on the Raspbian image from OVOS?

**Review by mikejgray — COMMENTED** [review]
Looks like good improvements overall. I did have one question about some dependencies being removed from the instructions.

File: docs/install_raspbian.md (hunk 1 above)
Comment (builderjer) [reply docs/install_raspbian.md:122]: yes, the image includes all of those packages

Comment (builderjer) [reply docs/install_raspbian.md:122]: and it is also in the `manual_user_install.sh` script.  Should not need it here

File: docs/install_raspbian.md [hunk 2]
```diff
@@ -1,154 +1,217 @@
...
+Check your installation with
 
-<strong>You should now have a running ovos device!!</strong>
+`systemctl --user status ovos-*`
 
-Check with this
+The full OVOS can take a few minutes to load (especially on a Pi 3), but the processes should all eventually say `active (running)`, except for `ovos.service` which should say `active (exited)`
 
-`systemctl --user status mycroft*`
+You can also track progress by watching /var/log/syslog.  Once things slow down you can try saying "Hey Mycroft". In a few seconds (the first time is slow) you should hear a 'ding' from the system. Then say "What day is it".  After a delay you should hear information about today's date.
```
Comment (builderjer) [inline docs/install_raspbian.md:207]: I would rather point to `~/.local/state/mycroft/<logfile.log>`  That is where they default to even without a systemd service file

**Review by builderjer — COMMENTED** [review]
One more change about the log files is all

File: docs/install_raspbian.md (hunk 2 above)
Comment (Bulwagga) [reply docs/install_raspbian.md]: I updated both references to syslog, changed it to refer users to ~/.local/state/mycroft/*.log

**Comment by Bulwagga** [comment]
Note I found and fixed another link that needed to be updated to point to new image server... See gs_installing_image.md.

**Review by builderjer — APPROVED** [review]

**Pull request merged**